### ISSUE TITLE
Broken link on live website

### ISSUE THREAD

**Issue by csharpforevermore** [getbem/getbem.github.io#210]
### Issue
(Note to self: My first day using BEM and I noticed a booboo so I am like woohoo!)
On [the BEM website's introduction page](http://getbem.com/introduction/) there are some links provided in bullet points under the heading **Further Reading**. The link is the 6th list item, recorded as [Used and Abused](http://www.phase2technology.com/blog/used-and-abused-css-inheritance-and-our-misuse-of-the-cascade/) — CSS Inheritance and Our Misuse of the Cascade. 
It seems that the article's authors, _Phase 2 Technology, LLC_ have renamed content on their site but without bothering with some (critical) 301 redirects to prevent Google from going postal on this website, and by proxy, the company itself. The 301 should point to the new article's home of https://www.phase2technology.com/blog/used-and-abused-css.

### TL;DR
BEM website [1] > **introduction** page > **further reading** section > List item hyperlink "Used and Abused — CSS Inheritance and Our Misuse of the Cascade" URL - update as follows:

* **before [2]:** https://www.phase2technology.com/blog/used-and-abused-css-inheritance-and-our-misuse-of-the-cascade/ 
* **after [3]:** **https://www.phase2technology.com/blog/used-and-abused-css**

![1](https://user-images.githubusercontent.com/4661508/100293670-57c65d00-2f7c-11eb-85bb-152fea9e399a.jpg)
![2](https://user-images.githubusercontent.com/4661508/100293673-585ef380-2f7c-11eb-9f03-42cf39de71af.jpg)
![3](https://user-images.githubusercontent.com/4661508/100293674-58f78a00-2f7c-11eb-9dc3-c520f2c3a37a.jpg)
